bin/PerlDDS/Run_Test.pm

```diff
@@ -830,7 +834,7 @@ sub _temporary_file_path {
     if ($flag) {
```

Comment:
Rather than passing a flag, you should check to see if target TEST_ROOT and TEST_FSROOT are set, as is done elsewhere.